easybuild/easyconfigs/s/ScaLAPACK/ScaLAPACK-2.0.2-gompi-1.6.20-OpenBLAS-0.2.13-LAPACK-3.5.0.eb

```diff
@@ -0,0 +1,29 @@
+name = 'ScaLAPACK'
+version = '2.0.2'
+
+homepage = 'http://www.netlib.org/scalapack/'
+description = """The ScaLAPACK (or Scalable LAPACK) library includes a subset of LAPACK routines
+ redesigned for distributed memory MIMD parallel computers."""
+
+toolchain = {'name': 'gompi', 'version': '1.6.20'}
+toolchainopts = {'pic': True}
+
+source_urls = [homepage]
+sources = [SOURCELOWER_TGZ]
+
+comp_name = 'GCC'
+comp_version = '4.8.4'
+comp = (comp_name, comp_version)
```

Comment:
no need for this trick here, since `comp` is only used in one place, replace use of `comp` with `('GCC', '4.8.4')`